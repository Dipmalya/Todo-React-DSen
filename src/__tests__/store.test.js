describe('Test suite for store', () => {
    it('Without dev tool', () => {
        process.env.NODE_ENV = 'development';
        const configStore = require('../store').default;
        const store = configStore();
        expect(store).toMatchSnapshot();
    })
})