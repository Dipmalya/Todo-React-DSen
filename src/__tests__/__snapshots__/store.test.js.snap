// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for store Without dev tool 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
